ra_aid/webui/server.py

```
@@ -0,0 +1,220 @@
+
+class ConnectionManager:
+    def __init__(self):
+        self.active_connections: List[WebSocket] = []
+
+    async def connect(self, websocket: WebSocket) -> bool:
+        try:
+            logger.debug("Accepting WebSocket connection...")
+            await websocket.accept()
+            logger.debug("WebSocket connection accepted")
+            self.active_connections.append(websocket)
+            return True
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)